docs/tools/mbed_cli_2/use.md

```diff
@@ -178,6 +178,12 @@ To perform an iterative build on a previously configured project:
 mbed-tools build
 ```
 
+## Flashing the built program
+You can flash the built program to the connected target by adding the -f/--flash argument to the compile command:
+```
+mbed-tools build -f
```

Comment:
should we use `compile` here, I'm assuming we are going to merge this PR after new Mbed tools 3.6.0 release

Reply:
Ah good point. Copy/paste error.